dusty/cli/assets.py

```diff
@@ -0,0 +1,44 @@
+"""
+Place special files in Dusty containers
+
+Assets are special files on your local filesystem, not located in
+repositories, that Dusty will put in containers for you.
```

Comment:
Let's try to clean up the terminology here

![](http://cdn.meme.am/instances/62533274.jpg)